src/main/resources/data/franken_errata/franken_errata.json

```diff
@@ -270,6 +270,18 @@
         "ItemId": "jolnar",
         "Undraftable": true
     },
+    {
```

Comment:
@newnew1450 which ruleset are you following that forces edict & imperia together? The one I know of from TTS/TTPG have them separated

Reply:
Oh I misremembered. Imperia gains the Commander, not Edict. I used this: https://docs.google.com/spreadsheets/d/1GWlcW1ue4oq-KwQCBBem8zF1LPX6a6QSxkP20jLmVu0/edit#gid=0

Reply:
@jamesrudolph how do I correct the update?

Reply:
Updated the change to Commander instead of Edict.

Reply:
I don't have access to that Google sheet, but I'll trust you lol